index.html

```diff
@@ -0,0 +1,234 @@
+
+          <h3>Lecture</h3>
+          <p>
+            Here, we will be in a
+            conference meeting
+            to get knowledge from experts
+          </p>
+        </div>
+
+        <div class="content1 d-flex">
+          <div class="nail">
+            <img src="./Images/forum.png" alt="">
+          </div>
+          <h3>forum</h3>
+          <p>
+            A chat forum will be created
+            for more info and ideas
+          </p>
+        </div>
+
+        <div class="content1 d-flex">
+          <div class="nail">
+            <img src="./Images/Connect.svg" alt="">
+          </div>
+          <h3>Connect</h3>
+          <p>
+            Connect with us globally and
+            extend your success chain
+          </p>
+        </div>
+
+        <div class="content1 d-flex">
+          <div class="nail">
+            <img src="./Images/Fitness.png" alt="">
+          </div>
+          <h3>Fitness</h3>
+          <p>
+            Remember, fitness stimulates
+            your brain and thus
+            good mindsets set.
+          </p>
+        </div>
+
+        <div class="content1 d-flex">
+          <div class="nail">
+            <img src="./Images/Goal.png" alt="">
+          </div>
+          <h3>Planning</h3>
+          <p>
+            Setting a goal with a specific
+            time of accomplishment is
+            a good start to succeed.
+          </p>
+        </div>
+
+      </div>
+
+      <a href="#">see the world program</a>
+      <button type="button" class="join">Join Celebrity Feud 2023</button>
+    </div>
+  </section>
+
+  <section class="featured-speakers">
+    <div class="head">
+      <h3>Featured Speakers</h3>
+      <div class="line"></div>
+    </div>
+    <div class="whole-featured-speakers" id="Spcann">
+
+    </div>
+    <button type="button" class="See-more-Btn" id="See-more-Btn">See More</button>
+  </section>
+
+  <footer>
+    <section class="site-logos">
+      <div class="head">
+        <h3>Partners</h3>
+        <div class="line"></div>
+      </div>
+      <div class="partners">
+        <img src="Images/adiddas (1).png" alt="">
+        <img src="Images/Chrysler.png" alt="">
+        <img src="Images/Amazon.svg" alt="">
+        <img src="Images/Samsung.png" alt="">
+        <img src="Images/tesla.png" alt="">
+      </div>
+
+    </section>
+
+    <!-- <section class="acknowledgement">
+            <div class="end-logo">
+                <img src="./Images/audi.png" alt="">
+            </div>
+            <div class="rights-reserved">
+                <h5><b>2023 @Wils FOURR speed. Some rights reserved.</b></h5>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Magnam voluptatum eum, impedit tenetur
+                    excepturi libero. Esse sequi<br> exercitationem illum, voluptatibus suscipit provident architecto.
+                    Animi dolore excepturi dolores amet fuga cumque.</p>
+            </div>
+        </section> -->
+
```

Comment:
Please remove unused pieces of code from your codebase in order to declutter it.